Multisig/Features/Security/SecurityCenter.swift

```diff
@@ -74,66 +74,106 @@ class SecurityCenter {
-    func `import`(id: DataID, ethPrivateKey: EthPrivateKey, completion: @escaping (Result<Bool?, Error>) -> ()) {
-        perfomSecuredAccess { [unowned self] result in
-            switch result {
-            case .success:
-                do {
-                    try sensitiveStore.import(id: id, ethPrivateKey: ethPrivateKey)
-                    completion(.success(true))
-                } catch let error {
-                    completion(.failure(GSError.KeychainError(reason: error.localizedDescription)))
+    func `import`(id: DataID, ethPrivateKey: EthPrivateKey, protectionClass: ProtectionClass = .sensitive, completion: @escaping (Result<Bool?, Error>) -> ()) {
+        switch(protectionClass) {
+        case .sensitive:
+            perfomSecuredAccess { [unowned self] result in
+                switch result {
+                case .success:
+                    do {
+                        try sensitiveStore.import(id: id, ethPrivateKey: ethPrivateKey)
+                        completion(.success(true))
+                    } catch let error {
+                        completion(.failure(GSError.KeychainError(reason: error.localizedDescription)))
+                    }
+                case .failure(let error):
+                    completion(.failure(error))
                 }
-            case .failure(let error):
-                completion(.failure(error))
+            }
+        case .data:
+            do {
+                try dataStore.import(id: id, ethPrivateKey: ethPrivateKey)
+                completion(.success(true))
+            } catch let error {
+                completion(.failure(GSError.KeychainError(reason: error.localizedDescription)))
             }
         }
     }
 
-    func remove(address: Address, completion: @escaping (Result<Bool?, Error>) -> ()) {
-        perfomSecuredAccess { [unowned self] result in
-            switch result {
-            case .success:
-                do {
-                    try sensitiveStore.delete(address: address)
-                    completion(.success(true))
-                } catch let error {
-                    completion(.failure(GSError.KeychainError(reason: error.localizedDescription)))
+    func remove(address: Address, protectionClass: ProtectionClass = .sensitive, completion: @escaping (Result<Bool?, Error>) -> ()) {
+        switch(protectionClass) {
+        case .sensitive:
+            perfomSecuredAccess { [unowned self] result in
+                switch result {
+                case .success:
+                    do {
+                        try sensitiveStore.delete(address: address)
+                        completion(.success(true))
+                    } catch let error {
+                        completion(.failure(GSError.KeychainError(reason: error.localizedDescription)))
+                    }
+                case .failure(let error):
+                    completion(.failure(error))
                 }
```

Comment:
Could we simplify it this way?
```suggestion
               do {
                    _ = try result.get()
                    try sensitiveStore.delete(address: address)
                    completion(.success(true))
                } catch {
                    completion(.failure(GSError.KeychainError(reason: error.localizedDescription)))
                }
```